### ISSUE TITLE
[Linux] Update Debian watch files to specify TIER

### ISSUE THREAD

**Issue by darcywong00** [keymanapp/keyman#2552]
Currently, the Debian watch files follow a pattern
```
https://downloads.keyman.com/linux/(?:beta|stable)/(\d\S+)/keyman_config-(.+)\.tar\.gz debian uupdate
```

But the uscan is only getting the latest `beta` release. This hampers `stable` releases from shipping to launchpad. From the uscan [documentation](https://manpages.debian.org/stretch/devscripts/uscan.1.en.html)
> (If multiple directories match, the highest version is picked.

### Affected files
/common/engine/keyboardprocessor/debian/watch
/linux/**/debian/watch

Need to investigate if `${TIER}` can be applied in the watch files...

**Comment by darcywong00** [comment]
As of 14.0 the tier will be set by the `./TIER.md` file

**Comment by darcywong00** [comment]
Fixed in 13.0.29 beta
